src/visualization/visalizeEpistasis/findSubsystemOfGenes.m

```diff
@@ -0,0 +1,153 @@
+function [rxns,subsys,subsysGenes,uSys,Nr,rxnNames]=findSubsystemOfGenes(model,genes,pseudoFlag)
```

Comment:
You might want to have a look at the `findSubSystemsFromGenes` function, which roughly does what this function is doing.